.eslintrc.js

```diff
@@ -320,16 +320,13 @@ module.exports = {
         /**
          * testing-library
          */
+        "testing-library/await-async-queries": "error",
         "testing-library/prefer-user-event": "error",
-        "testing-library/no-wait-for-empty-callback": "error",
-        "testing-library/no-wait-for-multiple-assertions": "error",
-        "testing-library/no-wait-for-side-effects": "error",
-        "testing-library/no-wait-for-snapshot": "error",
```

Comment:
Can you explain all of these removals?

Reply:
These are all enabled, by default, in the base `plugin:testing-library/react` ruleset: https://github.com/testing-library/eslint-plugin-testing-library?tab=readme-ov-file#supported-rules